cmd/crictl/util.go

```diff
@@ -210,7 +210,7 @@ func outputProtobufObjAsYAML(obj proto.Message) error {
 
 func outputStatusInfo(status string, info map[string]string, format string) error {
 	// Sort all keys
-	keys := make([]string, len(info))
+	var keys []string
```

Comment:
I think something like this should fix the linter warning:
```suggestion
keys := []string{}
```

Reply:
yep 

```
cmd/crictl/util.go:213:2: Consider preallocating `keys` (prealloc)
	var keys []string
	^
Makefile:92: recipe for target 'lint' failed
make: *** [lint] Error 1
The command "make lint" exited with 2.
```

Reply:
Done